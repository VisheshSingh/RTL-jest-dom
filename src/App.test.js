import React from 'react';
import { render } from '@testing-library/react';
import App from './App';

describe('Testing App Component', () => {
  test('Should render App component with any errors', () => {
    const { asFragment } = render(<App />);
    expect(asFragment(<App />)).toMatchSnapshot();
  });
});
