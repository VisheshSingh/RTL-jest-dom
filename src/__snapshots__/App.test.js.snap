// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing App Component Should render App component with any errors 1`] = `
<DocumentFragment>
  <div
    class="App"
  >
    <h1>
      Testing Updated
    </h1>
  </div>
</DocumentFragment>
`;
